others/_posts/2017-09-18-sweat-and-toil.md

```diff
@@ -7,8 +7,8 @@ redirect_from: "/child-labor/"
 ---
 These datasets contain information on child labor and forced labor worldwide from ILAB’s three flagship 
 reports: <i>Findings on the Worst Forms of Child Labor</i>; <i>List of Goods Produced by Child Labor or Forced Labor</i>
-; and <i>List of Products Produced by Forced or Indentured Child Labor</i>. There are 14 tables containing data from the 2015-2018 reporting cycles and 11 tables from the 2014 reporting cycle. This information is also available in ILAB’s app, Sweat & Toil: Child Labor, 
-Forced Labor, and Human Trafficking Around the World. For more information, see ILAB’s <a href="https://www.dol.gov/agencies/ilab/resources/reports/child-labor">International Child Labor and Forced Labor Reports page</a>.
+; and <i>List of Products Produced by Forced or Indentured Child Labor</i>. There are 14 tables containing data from the 2015-2019 reporting cycles and 11 tables from the 2014 reporting cycle. This information is also available in ILAB’s app, Sweat & Toil: Child Labor, 
+Forced Labor, and Human Trafficking Around the World. For more information, see ILAB’s <a href="https://www.dol.gov/agencies/ilab/resources/reports/child-labor">International Child Labor and Forced Labor Reports page</a>. Please cite USDOL when using  Sweat & Toil data.
```

Comment:
Looks like we have extra space in "Please cite USDOL when using Sweat & Toil data."